lib-injection/README.md

```diff
@@ -2,14 +2,29 @@
 
 ## What is the lib-injection feature?
 
-The lib-injection project is a tool to allow injection of the tracer into a customer's application container without
-requiring them to modify their application images.
+The lib-injection project is a feature to allow injection of the Datadog library
```

Comment:
We should update the entire document...

- lib-injections tests functions
- How to run the lib-injection tests in CI
- How to run the lib-injection tests in Local

It is important that the scripts can be easily executed locally.

Reply:
> How to run the lib-injection tests in CI

The auto mode is only meant to run here in system-tests, the manual one is already documented

> How to run the lib-injection tests in Local

> It is important that the scripts can be easily executed locally.

Extended the instructions on how to run locally to include the auto mode, it's pretty straightforward! 

> We should update the entire document...
lib-injections tests functions

Could you elaborate? Are you asking to add doc strings to the functions in `functions.sh` ? Or is it about something else?